src/mumble/Markdown.cpp

```diff
@@ -312,7 +312,7 @@ bool processPlainLink(QString &str, int &offset) {
 	// We support links with prefixed protocol (e.g. https://bla.com) and prefixed with www (e.g. www.bla.com)
 	// See also https://stackoverflow.com/a/1547940/3907364
 	static const QRegularExpression s_regex(
-		QLatin1String("([a-zA-Z]+://|[wW][wW][wW]\\.)[A-Za-z0-9-._~:/?#\\[\\]@!$&'()*+,;=]+"));
+		QLatin1String("([a-zA-Z]+://|[wW][wW][wW]\\.)[A-Za-z0-9-._~:/?#\\[\\]@!$&%'()*+,;=]+"));
```

Comment:
Given that percent may only be used in an escape sequence that starts by `%` followed by a two digit hex-number, I think it might be better to do something like this:
```suggestion
		QLatin1String("([a-zA-Z]+://|[wW][wW][wW]\\.)([A-Za-z0-9-._~:/?#\\[\\]@!$&'()*+,;=]|%[a-fA-F0-9]{2})+"));
```

Reply:
Done